.github/workflows/_deploy_kafka_test.yml

```diff
@@ -83,7 +89,10 @@ jobs:
           az aks get-credentials --name $CLUSTER_NAME --resource-group $RESOURCE_GROUP  --overwrite-existing
           cd main/kafka
           chmod +x test-setup.sh
-          bash test-setup.sh kafka-demo-pipeline $AZURE_AKV_RESOURCE_ENDPOINT
+          randomid=$(openssl rand -hex 8 | tr -d '\n')  
```

Comment:
On @hgarvison other branch, we also switched this to use the git commit hash or job ID instead of a random number.  Use the commit hash so that we can track that value between different runs.